kafka/Dockerfile.ubi8

```diff
@@ -73,8 +73,8 @@ enabled=1 " > /etc/yum.repos.d/confluent.repo \
     && rm -rf /tmp/* \
     && echo "===> Setting up ${COMPONENT} dirs" \
     && mkdir -p /var/lib/${COMPONENT}/data /etc/${COMPONENT}/secrets \
-    && chown appuser:appuser -R /var/lib/${COMPONENT} /etc/${COMPONENT} \
-    && chmod -R ag+w /etc/kafka /var/lib/${COMPONENT}/data /etc/${COMPONENT}/secrets
+    && chmod -R ag+w /etc/kafka /var/lib/${COMPONENT}/data /etc/${COMPONENT}/secrets \
+    && chown -R appuser:appuser /var/log/kafka /var/log/confluent /var/lib/kafka /var/lib/zookeeper /etc/${COMPONENT}/secrets
```

Comment:
@sdanduConf Any reason to stop using the COMPONENT env variable here? 
Was there also a permissions issue with /var/log/confluent?
Same question about /var/lib/zookeeper?
Why did we decide to not change the ownership of the entire /etc/${COMPONENT} folder and instead just the secrets folder with in that folder?